src/sage/structure/sequence.py

```diff
@@ -228,8 +233,8 @@ def Sequence(x, universe=None, check=True, immutable=False, cr=False, cr_str=Non
                     x = x.gens()
 
     if universe is None:
-        orig_x = x
-        x = list(x)  # make a copy even if x is a list, we're going to change it
+        orig_x = list(x)
```

Comment:
```suggestion
        if not isinstance(x, (list, tuple)):
            orig_x = list(x)
        else:
            orig_x = x
```
No need to make a second copy if it already is a list. The `isinstance` check should be faster in nearly all cases than creating the shallow copy.